framework/doc/content/source/meshgenerators/StitchedMeshGenerator.md

```diff
@@ -15,6 +15,8 @@ Consider the following three meshes.
 Using the `StitchedMeshGenerator` object from within the [Mesh](/Mesh/index.md) block of the input file, as shown in the input
 file snippet below, these three square meshes are joined into a single mesh as shown in Figure 4.
 
+Note that the way that the meshes are merged gives precedence to the left-most mesh listed in terms of sidesets: the sidesets of the second, third, etc meshes will be subsumed into the sidesets of the first mesh. The names of the sidesets in the first mesh are what the names that will remain in the outputted mesh. 
```

Comment:
```suggestion
Note that the way that the meshes are merged gives precedence to the left-most mesh listed in terms of sidesets: the sidesets of the second, third, etc meshes will be absorbed into the sidesets of the first mesh. The names of the sidesets in the first mesh are what the names will remain in the outputted mesh. 
```

Reply:
Neither of us did that last sentence justice. It should be "The names of the sidesets in the first mesh are what names will remain in the outputted mesh. "

Thanks for the other suggestion. Will implement.